addons/website/views/snippets/snippets.xml

```diff
@@ -1250,49 +1250,120 @@
-            <we-button data-select-class="o_anim_shake">Shake</we-button>
-            <we-button data-select-class="o_anim_tada">Tada</we-button>
-            <we-divider/>
-            <we-button data-select-class="o_anim_flip_in_x">Flip-In-X</we-button>
-            <we-button data-select-class="o_anim_flip_in_y">Flip-In-Y</we-button>
+        <!-- Animation mode -->
+        <we-row>
+            <we-select string="Animation" data-dependencies="!image_transform_opt"
+                       data-is-animation-type-selection="true" data-no-preview="true">
+                <we-button data-animation-mode=""
+                           data-select-class=""
+                           data-name="no_animation_opt">None</we-button>
+                <we-button data-animation-mode="onAppearance"
+                           data-force-animation="true"
+                           data-select-class="o_animate"
+                           data-name="animation_on_appearance_opt">On Appearance</we-button>
+                <we-button data-animation-mode="onScroll"
+                           data-select-class="o_animate o_animate_on_scroll"
+                           data-force-animation="true"
+                           data-name="animation_on_scroll_opt">On Scroll</we-button>
+            </we-select>
+            <we-button-group data-dependencies="animation_on_scroll_opt"
+                             data-no-preview="true" data-force-animation="true">
+                <we-button data-select-class="">In</we-button>
+                <we-button data-select-class="o_animate_out">Out</we-button>
+            </we-button-group>
+        </we-row>
+        <!-- Effect -->
+        <we-select string="Effect" class="o_we_sublevel_1" data-dependencies="!no_animation_opt">
+            <we-button data-select-class=""
+                       data-name="o_anim_no_effect_opt"
+                       data-dependencies="no_animation_opt">None</we-button>
+            <we-button data-select-class="o_anim_fade_in"
+                       data-trigger="animation_direction_in_place_opt">Fade</we-button>
+            <we-button data-select-class="o_anim_slide_in"
+                       data-name="o_anim_slide_in_opt"
+                       data-trigger="animation_direction_from_right_opt">Slide</we-button>
+            <we-button data-select-class="o_anim_bounce_in"
+                       data-trigger="animation_direction_in_place_opt">Bounce</we-button>
+            <we-button data-select-class="o_anim_rotate_in"
+                       data-name="o_anim_rotate_opt"
+                       data-trigger="animation_direction_in_place_opt">Rotate</we-button>
+            <we-button data-select-class="o_anim_zoom_out"
+                       data-trigger="animation_direction_in_place_opt">Zoom Out</we-button>
+            <we-button data-select-class="o_anim_zoom_in"
+                       data-trigger="animation_direction_in_place_opt">Zoom In</we-button>
```

Comment:
Just wondering: do we really need the data-trigger ?

Reply:
yes because of the `direction` options of `rotate` which are different, we could avoid that but it should be done in .js then (for example, if old `animation` is `rotate` and old `direction` is `o_anim_from_top_right` then new `animation` must have `direction` => `o_anim_from_right`) But it doesn't maybe not worth it